### PR TITLE
Added E2E Test for Grading Standards

### PR DESCRIPTION
**Pull request by davidvarga93**
refs: MBL-16088
affects: Student
release note: none

test plan:

### CODE REVIEW

**Comment by inst-danger** [comment]
<details><summary>Student.ipa</summary>
  <a href="https://app.bitrise.io/app/88a3641e29e4745c/build/866911cc-f1df-4101-ae4a-d2765c329eb7/artifact/ff9431b6b40c1d55/p/a05105d34919870ada138c5311ac71fb">
    
  </a>
</details>
<details><summary>Teacher.ipa</summary>
  <a href="https://app.bitrise.io/app/88a3641e29e4745c/build/866911cc-f1df-4101-ae4a-d2765c329eb7/artifact/cedc7722c81fff16/p/0d98d65481dc2eb66f6b3d802083f3b0">
    
  </a>
</details>
<details><summary>Parent.ipa</summary>
  <a href="https://app.bitrise.io/app/88a3641e29e4745c/build/866911cc-f1df-4101-ae4a-d2765c329eb7/artifact/606ee68927d3305b/p/d1ce1804e12f0523dc926513b1476aba">
    
  </a>
</details>

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


<details><summary>:x: Swift lint</summary>


    ❌ /Users/vagrant/git/TestsFoundation/TestsFoundation/DataSeeding/Submissions/Grades/GradingStandard/CreateDSGradingStandardsRequest.swift:44:89: Trailing Comma Violation: Multi-line collection literals should have trailing commas. (trailing_comma)
    /Users/vagrant/git/Student/StudentE2ETests/Grades/DSGradingStandardsE2ETests.swift:58:1: error: Vertical Whitespace Violation: Limit vertical whitespace to a single empty line. Currently 2. (vertical_whitespace)
    
</details>


[MBL-16088](https://instructure.atlassian.net/browse/MBL-16088)

#### Affected Apps: Student
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by davidvarga93** [comment]
Screenshot 2022-06-07 at 9 40 35

File: TestsFoundation/TestsFoundation/DataSeeding/Submissions/Grades/GradingStandard/CreateDSGradingStandardsRequest.swift [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        self.path = "courses/\(courseId)/grading_standards"
+    }
+}
+
+extension CreateDSGradingStandardsRequest {
+    public struct RequestDSGradingStandards: Encodable {
+        public let title: String
+        public let grading_scheme_entry: [GradeEntry]
+        public static let StandardGradingScheme: [GradeEntry] = [
+            GradeEntry.init(name: "A", value: 94), GradeEntry.init(name: "A-", value: 90),
```
Comment (vargaat) [inline TestsFoundation/TestsFoundation/DataSeeding/Submissions/Grades/GradingStandard/CreateDSGradingStandardsRequest.swift:39]: `.init`s could be omitted.

File: TestsFoundation/TestsFoundation/DataSeeding/Submissions/Grades/GradingStandard/CreateDSGradingStandardsRequest.swift [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import Core
+
+public class CreateDSGradingStandardsRequest: APIRequestable {
+    public typealias Response = DSGradingStandard
+
+    public let method = APIMethod.post
+    public var path: String
```
Comment (vargaat) [inline TestsFoundation/TestsFoundation/DataSeeding/Submissions/Grades/GradingStandard/CreateDSGradingStandardsRequest.swift:25]: Is this modified from anywhere? If not, we could use `let` instead.

File: TestsFoundation/TestsFoundation/DataSeeding/Course/CreateDSCourseRequest.swift [hunk 3]
```diff
@@ -40,5 +40,35 @@ extension CreateDSCourseRequest {
...
         let course: RequestedDSCourse
         let offer = true // makes the course published after creation
+
+        public init(course: RequestedDSCourse) {
+            self.course = course
+        }
+    }
+}
+
+struct UpdateDSCourseRequest: APIRequestable {
```
Comment (vargaat) [inline TestsFoundation/TestsFoundation/DataSeeding/Course/CreateDSCourseRequest.swift:50]: If these are about updates would it make sense to create an `UpdateDScourseRequest.swift` file for them?

**Review by vargaat — CHANGES_REQUESTED** [review]

**Review by vargaat — APPROVED** [review]

**Pull request merged**